style.css

```diff
@@ -0,0 +1,258 @@
+@import url('https://fonts.googleapis.com/css?family=Lexend+Deca|Quicksand&display=swap');
```

Comment:
Other than some heights and width difference and color, this is pretty spot on. Good use of flex to accommodate for resizing browser size just like the real site. Also I like the use of icons for Explore, Bookmark part of the page. Keep up the good work!

Reply:
Thanks!